requirements.txt

```diff
@@ -1,10 +1,11 @@
-# from `pip freeze` under Python 3.6 lambda image 16-Apr-2020
-boto3==1.12.22
-botocore==1.15.22
-Cython==0.29.21
-docutils==0.15.2
-jmespath==0.9.5
-python-dateutil==2.8.1
-s3transfer==0.3.3
-six==1.14.0
-urllib3==1.25.8
+# from `pip freeze` under Python 3.8 lambda image 01-Sep-2021
+boto3==1.17.100
+botocore==1.20.100
+Cython==0.29.24
+jmespath==0.10.0
+python-dateutil==2.8.2
+rapid-client==0.0.0
```

Comment:
This looks sketchy? No information at all about that package.

Reply:
That's what I get for trusting Amazon. They install it. But on PyPI this is one of the sketchiest packages I've ever seen https://pypi.org/project/rapid-client/#files. The Homepage is example.com! WTH?
```
b'boto3==1.17.100\nbotocore==1.20.100\njmespath==0.10.0\npython-dateutil==2.8.2\nrapid-client==0.0.0\ns3transfer==0.4.2\nsix==1.16.0\nurllib3==1.26.6\n'
```
I'm gonna report this to AWS.